Wrappers/Python/ccpi/viewer/utils/conversion.py

```diff
@@ -503,13 +487,10 @@ def WriteNumpyAsMETAImage(array, filename, spacing=(1., 1., 1.), origin=(0., 0.,
         else:
             numpy.save(datafname, numpy.asfortranarray(array))
         npyhdr = parseNpyHeader(datafname)
-
-        typecode = array.dtype.char
-        print("typecode,", typecode)
-        #r_type = Converter.numpy_dtype_char_to_MetaImageType[typecode]
+        typecode = str(array.dtype)
         big_endian = 'True' if npyhdr['description']['descr'][0] == '>' else 'False'
-        readshape = descr['description']['shape']
-        is_fortran = descr['description']['fortran_order']
+        readshape = npyhdr['description']['shape']
```

Comment:
I'm not really proud of the variable name `readshape`. It means _shape that has been read_ . It doesn't matter much, but it's not so clear.

Reply:
okay, I'll be changing this method in another PR relating to #196 so I'll make sure to update that there